src/modules/function/ret.rs

```diff
@@ -56,12 +56,15 @@ impl SyntaxModule<ParserMetadata> for Return {
 
 impl TranslateModule for Return {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
-        let (name, id, variant) = meta.fun_name.clone().expect("Function name not set");
+        let fun_name = meta.fun_meta.as_ref()
```

Comment:
Missed a rename here

Reply:
No I didn't; read the full function chain:
```
let fun_name = meta.fun_meta.as_ref()
    .expect("Function name not set")
    .mangled_name();
```

Reply:
I have refactored this to make it a bit more obvious:
```
let fun_name = meta.fun_meta.as_ref()
    .map(FunctionMetadata::mangled_name)
    .expect("Function name not set");
```